probeengine/probe_helpers_test.go

```diff
@@ -117,34 +110,29 @@ func TestGetFeaturePath(t *testing.T) {
 }
 
 func Test_getTmpFeatureFile(t *testing.T) {
+	config.GlobalConfig.TmpDir = t.TempDir()
+	filename := "Test_getTmpFeatureFile.feature"
+	os.Create(filepath.Join(testFolder(), filename))
+	os.MkdirAll(filepath.Join(config.GlobalConfig.TmpDir, "probeengine", "testdata"), 0755)
```

Comment:
Admittedly, the test changes here got a bit cumbersome. I'm hoping to spend some time improving and extending these tests very soon.